scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixEvaluation.java

```diff
@@ -0,0 +1,32 @@
+package scalafix.interfaces;
+
+import java.util.Optional;
+
+public interface ScalafixEvaluation {
+    /**
+     * @return boolean true if scalafix has successfully been evaluated on all files configured.
+     */
+    boolean isSuccessful();
+
+    /**
+     * @return List of scalafixErrors resulted from scalafix evaluation
+     */
+    ScalafixError[] getErrors();
+
+    /**
+     * @return A message error is provided if the run is not successful
+     */
+    Optional<String> getMessageError();
```

Comment:
how does this relate to the `getErrors` ?

Reply:
It's a global message error that can be empty. Errors are a concatenation of code errors that result from a merge function.